lib/modules/accountSwitcher.js

```diff
@@ -434,12 +456,42 @@ function _notifySwitchedAccountElsewhere() {
 		message,
 	});
 }
-
+function autoSwitcher(thing) {
+	if(thing.getAuthor() != loggedInUser() && window.location.search.indexOf('accountSwitcher=auto') == -1 ) {
```

Comment:
DRY:

`const autoSwitched = window.location.search.indexOf('accountSwitcher=auto') === -1;`

then later  `if (autoSwitched && ...)` or `if (!autoSwitched && ...)`

Reply:
Alternatively, `const autoSwitched = _.once(() =>  window.location.search.indexOf('accountSwitcher=auto') === -1;`  and call `if (autoSwitched() && ...`